### ISSUE TITLE
Dev: Replace Dash with Pipe in Any Overlay With Text "Clear"

### ISSUE THREAD

**Issue by Rabia2219** [hackforla/tdm-calculator#3372]
### Overview
We need to replace the dash with a pipe in all overlays across the app that have the text "Clear" so that screen readers interpret the linked options correctly

Note: Feedback page does not have the column header, therefore no overlays

#### Details
- ID (appears on My Projects, Manage Submissions, Feedback, Submissions, My Projects Admin)
   <details><summary>Before - ID</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/d99c3029-991e-4c17-8129-7d6e639ddd41" />
   </details> 
- Project Name (appears on My Projects, Manage Submissions, Feedback, Submissions, My Projects Admin)
   <details><summary>Before - Project Name in My Projects</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/b34f805d-dc31-43f6-a899-24b01aceb7cc" />
   </details> 
   <details><summary>Before - Project Name in Manage Submissions, Default Sort is Applied</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/158f19d8-c79b-47a7-bcbe-f12f91385c90" />
   </details> 
   <details><summary>Before - Project Name in Submissions, Default Sort is Applied</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/951e8e87-be22-4cda-b816-d5aec32c5ca7" />
   </details> 
- Address (appears on My Projects, Feedback, Submissions, My Projects Admin)
   <details><summary>Before - Address</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/b2e4680f-e55e-498b-9ffa-efe798a4000a" />
   </details>
- Alt No. (appears on My Projects, My Projects Admin)
   <details><summary>Before - Alt No.</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/3f00eb00-c4fd-4560-8714-f4db72b1f9b5" />
   </details>
- Created By (appears on My Projects, Manage Submissions, My Projects Admin)
   <details><summary>Before - Created By</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/65018b2b-c94b-4393-83c0-e2e1c6da950e" />
   </details>
- DRO (appears on My Projects, My Projects Admin, Manage Submissions, Submissions)
   <details><summary>Before - My Projects, DRO</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/c61dad41-c224-4476-a21e-dea8d6d76bc8" />
   </details>
   <details><summary>Before - Manage Submissions and Submissions, DRO</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/e6117805-3e42-402d-bf1a-da97e9ce8189" />
   </details> 
- Admin Notes (appears on My Projects Admin, Manage Submissions)
   <details><summary>Before - Admin Notes</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/83f72d73-68c9-42eb-9ab9-4fe1ac3da9d6" />
   </details>
- Guidelines Version (appears on My Projects (Not Yet Implemented), My Projects Admin)
   <details><summary>Before - Guidelines Version</summary>
   </details>
- Level (appears on Manage Submissions, Submissions)
   <details><summary>Before - Level</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/13dc870a-8785-4611-9726-2310a125977f" />
   </details>
- Assignee (appears on Manage Submissions) / Staff Assigned (appears on Submissions) - the column name will be rationalized after talking to stakeholders #3375
   <details><summary>Before - Assignee, Manage Submissions</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/3e5ca2af-6140-44d4-8208-2a40d2e6d598" />
   </details> 
   <details><summary>Before - Staff Assigned, Submissions</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/fa01f423-4518-4893-9e6f-f9b23efdc9a4" />
   </details> 
- Invoice Status (appears on Manage Submissions, Submissions)
   <details><summary>Before - Invoice Status</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/fac6c66c-f8da-421f-afa1-ac68ac9d64bd" />
   </details>
- Submission Status (appears on Manage Submissions, Submissions)
   <details><summary>Before - Submission Status</summary>
   <img width="200" alt="Image" src="https://github.com/user-attachments/assets/d3a3418f-ebcf-4de1-8726-b5951f9eaf03" />
   </details>

<details><summary>Before (current) - Replace Dash with Pipe on All Overlays with Clear</summary>
<img width="200" alt="Image" src="https://github.com/user-attachments/assets/33156d85-421a-44bd-ab08-992b89858521" />
</details> 

<details><summary>After - Replace Dash with Pipe on All Overlays with Clear</summary>
<img width="200" alt="Image" src="https://github.com/user-attachments/assets/2928c627-d1d7-4c27-80ce-d42b6b80db38" />
</details> 

### Action Items
- [ ] Assign yourself to the development issue.
- [ ] Move the development issue from the Prioritized Backlog to In Progress.
- [ ] Implement the requested changes
- [ ] Create a PR, if applicable.
- [ ] Create a new slide (or edit one if it already exists) in staging and add the before and after images
- [ ] Once the PR has been approved, use it to close this issue. If there is no PR is associated with this issue, close this issue after lead signoff.

### Resources/Instructions
- Design mockups: [Figma, Full Design System, Filter Overlays, Filter Overlay Components](https://www.figma.com/design/nD9QK56Mzq7xNSaSUoeGx0/TDM-Calculator?node-id=33071-21187&t=NwNdCy5Qdf6iBfhs-4)

#### Staging and Release decks
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/)
   - ~[Staging slide, 3372](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3f98872dc86_0_42#slide=id.g3f98872dc86_0_42)~
- Release Deck
   - [2026-08-18 Release deck, slide 3372](https://docs.google.com/presentation/d/16-zgcjRJ6zjrAeTPZatTKdOAhuyhhqiN6bPsCsp0Huc/edit?slide=id.g3f6ead8bc21_6_126#slide=id.g3f6ead8bc21_6_126)

**Comment by Rabia2219** [comment]
### Error when clicking on Guidelines Version column filter icon
<img width="800" alt="Image" src="https://github.com/user-attachments/assets/dfb07505-6e9d-4f48-b022-1f5dcbd7aa0e" />

This will be resolved when this issue is merged
- https://github.com/hackforla/tdm-calculator/issues/3108
